tsdb/db.go

```diff
@@ -1074,6 +1074,17 @@ func (db *DB) Dir() string {
 	return db.dir
 }
 
+func (db *DB) ListBlockMetas() []BlockMeta {
```

Comment:
Both in Go and in Prometheus, the naming conventions for such list / getter functions is usually to omit the "List..." / "Get..." prefix, see also:

* https://github.com/prometheus/prometheus/blob/bfbae39931a6ddeb5913a6ea8d48a34bbebc6d29/rules/manager.go#L378-L379
* https://github.com/prometheus/prometheus/blob/bfbae39931a6ddeb5913a6ea8d48a34bbebc6d29/scrape/manager.go#L358-L359
* https://github.com/prometheus/prometheus/blob/bfbae39931a6ddeb5913a6ea8d48a34bbebc6d29/scrape/manager.go#L370-L371
* etc.

So I'd call this `BlockMetas()`.